packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -1,4 +1,65 @@
+const listOfMetricsSend = new Set()
+
+function debounce(fn, timeout) {
+  let timer = null
+
+  return function (...args) {
+    if (timer) {
+      clearTimeout(timer)
+    }
+
+    timer = setTimeout(fn, timeout, ...args)
+  }
+}
+
+function sendWebVitals(dataLayerName = `dataLayer`) {
+  const win = window
```

Comment:
just cleaning up stuff that is not really used in GA
```suggestion
function sendWebVitals() {
```